route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -4847,6 +4849,28 @@ public bool RemoveOrders(string[] orderIds, out string errorString)
             return GetJsonObjectFromAPIAsync<DataObject>(request, R4MEInfrastructureSettings.ApiHost, HttpMethodType.Put, null, false, false);
         }
 
+        /// <summary>
+        ///     Transfer an order to other organization
+        /// </summary>
+        /// <param name="transferredOrder"> An order to transfer </param>
+        /// <param name="organizationApiKey">An API key of a destination organization</param>
+        /// <returns>Transferred order</returns>
+        public Order TransferOrderToOrganization(Order transferredOrder, string organizationApiKey, out string errorString)
```

Comment:
Is it possible to add test for that functionality?

Reply:
Yes, it's possible, but in this case we need to expose 2 API keys - one of the current prime user, other - for destination organization and this is problem for me - I have access only to one test account and before uploading to the GitHub repo, I'm changing it with demo API key 111.... (which doesn't work)